packages/react/src/TextInput/TextInput.tsx

```diff
@@ -149,12 +169,14 @@ const TextInput = React.forwardRef<HTMLInputElement, TextInputProps>(
           aria-required={required}
           aria-invalid={validationStatus === 'error' ? 'true' : undefined}
           {...inputProps}
+          aria-describedby={inputDescribedBy}
           data-component="input"
         />
         <TextInputInnerVisualSlot
           visualPosition="trailing"
           showLoadingIndicator={showTrailingLoadingIndicator}
           hasLoadingIndicator={typeof loading === 'boolean'}
+          id={trailingVisualId}
```

Comment:
Since the leading and trailing visuals are now accessible as part of the description, should they be hidden from screen readers as part of the normal document flow? Ie, should we add `aria-hidden`?

Reply:
Yup, I think that's fair!